critiquebrainz/frontend/views/artist.py

```diff
@@ -1,25 +1,28 @@
 from collections import OrderedDict
 from flask import Blueprint, render_template, request, redirect, url_for
+from flask_login import current_user
 from flask_babel import gettext
 from werkzeug.exceptions import BadRequest, NotFound
 import critiquebrainz.db.review as db_review
 import critiquebrainz.frontend.external.musicbrainz_db.artist as mb_artist
 import critiquebrainz.frontend.external.musicbrainz_db.release_group as mb_release_group
 import critiquebrainz.frontend.external.musicbrainz_db.exceptions as mb_exceptions
-from critiquebrainz.frontend.views import BROWSE_RELEASE_GROUPS_LIMIT, ARTIST_REVIEWS_LIMIT
+from critiquebrainz.frontend.views import get_avg_rating, BROWSE_RELEASE_GROUPS_LIMIT, ARTIST_REVIEWS_LIMIT
+from critiquebrainz.frontend.forms.rate import RatingEditForm
+
 
 artist_bp = Blueprint('artist', __name__)
 
 
-@artist_bp.route('/<uuid:mbid>')
-def entity(mbid):
+@artist_bp.route('/<uuid:id>')
+def entity(id):
```

Comment:
👍  good for consistency, but let's really check that our tests make sure we didn't miss anything!

Reply:
Indeed, I did another pass over the code and found at some places we were still using mbid. I think all of the occurences are now replaced 🤞.